monitor/dbt_project/macros/get_new_model_alerts.sql

```diff
@@ -1,19 +1,24 @@
 {% macro get_new_model_alerts(days_back) %}
     -- depends_on: {{ ref('alerts_models') }}
     {% set select_new_alerts_query %}
-        WITH alerts AS (
-            SELECT * FROM {{ ref('alerts_models') }}
-            WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
+        with alerts as (
+            select * from {{ ref('alerts_models') }}
+            where alert_sent = false and detected_at >= {{ get_alerts_time_limit(days_back) }}
         ),
-
-        models AS (
-            SELECT * FROM {{ ref('elementary', 'dbt_models') }}
+        models as (
+            select * from {{ ref('elementary', 'dbt_models') }}
+        ),
+        snapshots as (
+            select * from {{ ref('elementary', 'dbt_snapshots') }}
```

Comment:
dbt_snapshot is a new model, so accessing it here means that the CLI won't work with old dbt package versions. I think that you need to first check if the table exists using the get_relation